libs/sdk-core/src/breez_services.rs

```diff
@@ -324,6 +324,21 @@ impl BreezServices {
         self.persister.get_node_state()
     }
 
+    /// Sign given message with privkey
+    pub async fn sign_message(&self, message: String) -> Result<String> {
```

Comment:
Recently we started shifting towards using structs for request response, e.g:
SignMessageRequest
SignMessageResponse
It makes it easier to maintain backward compatibility. Could you please use this approach?